includes/wc-order-functions.php

```diff
@@ -15,6 +15,165 @@
 }
 
 /**
+ * Wrapper for get_posts specific to orders.
+ *
+ * This function should be used for order retrieval so that when we move to
+ * custom tables, functions still work.
+ *
+ * Args:
+ * 		status array|string List of order statuses to find
+ * 		type array|string Order type, e.g. shop_order or shop_order_refund
+ * 		parent int post/order parent
+ * 		customer int|string|array User ID or billing email to limit orders to a
+ * 			particular user. Accepts array of values. Array of values is OR'ed. If array of array is passed, each array will be AND'ed.
+ * 			e.g. test@test.com, 1, array( 1, 2, 3 ), array( array( 1, 'test@test.com' ), 2, 3 )
+ * 		limit int Maximum of orders to retrieve.
+ * 		offset int Offset of orders to retrieve.
+ * 		page int Page of orders to retrieve. Ignored when using the 'offset' arg.
+ * 		exclude array Order IDs to exclude from the query.
+ * 		orderby string Order by date, title, id, modified, rand etc
+ * 		order string ASC or DESC
+ * 		return string Type of data to return. Allowed values:
+ * 			ids array of order ids
+ * 			objects array of order objects (default)
+ * 		paginate bool If true, the return value will be an array with values:
+ * 			'orders'        => array of data (return value above),
+ * 			'total'         => total number of orders matching the query
+ * 			'max_num_pages' => max number of pages found
+ *
+ * @since  2.6.0
+ * @param  array $args Array of args (above)
+ * @return array|object Number of pages and an array of order objects if
+ *                             paginate is true, or just an array of values.
+ */
+function wc_get_orders( $args ) {
+	$args = wp_parse_args( $args, array(
+		'status'   => array_keys( wc_get_order_statuses() ),
+		'type'     => wc_get_order_types( 'view-orders' ),
```

Comment:
post_type supports array or string, and this should support future order types so I think this one is safe to leave as-is